playbooks/roles/redis-exporter/meta/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+  dependencies:
+    - name: prometheus_redis_exporter_role
+      redis_exporter_redis_addr: "{{ REDIS_EXPORTER_REDIS_ADDR }}"
+      redis_exporter_check_keys: "{{ REDIS_EXPORTER_CHECK_KEYS }}"
+    - name: nginx-proxy
+      NGINX_PROXY_HTPASSWD_PATH: "{{ REDIS_EXPORTER_HTPASSWD_PATH }}"
+      # The username is a bit misleading – should actually be "prometheus", since it's the server
+      # authenticating, and the same username is used for scraping other exporters, but it's not worth
+      # the hassle to change it.
+      NGINX_PROXY_USERNAME: node-exporter
+      NGINX_PROXY_PASSWORD: "{{ REDIS_EXPORTER_PASSWORD }}"
+      NGINX_PROXY_AUTH_DOMAIN: Prometheus Redis Exporter
+      NGINX_PROXY_CONFIG_PATH: "{{ REDIS_EXPORTER_NGINX_CONFIG_PATH }}"
+      NGINX_PROXY_PUBLIC_PORT: 19121
```

Comment:
@shimulch, @0x29a, this port should be opened in the firewall using `ufw`. To avoid adding new ports for each service, is it possible to use virtual hosts listening on the same port to serve multiple sites?

Reply:
@lgp171188 I think we can make use of ``NGINX_PROXY_SERVER_NAME`` config from ``nginx-proxy`` role. But the server name might need to be passed from ``secrets`` repository, just like Ocim domain names get passed down.